packages/scaffolder-cli/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "@scaffolder/cli",
+  "version": "0.1.0-next.0",
+  "description": "CLI for local development of Software templates",
+  "backstage": {
+    "role": "cli"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "keywords": [
+    "backstage",
+    "scaffolder"
+  ],
+  "homepage": "https://backstage.io",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/backstage/backstage",
+    "directory": "packages/scaffolder-cli"
+  },
+  "license": "Apache-2.0",
+  "main": "dist/index.cjs.js",
+  "bin": "bin/scaffolder-cli",
+  "files": [
+    "asset-types",
+    "templates",
+    "config",
+    "bin",
+    "dist/**/*.js"
+  ],
+  "scripts": {
+    "build": "backstage-cli package build",
+    "clean": "backstage-cli package clean",
+    "lint": "backstage-cli package lint",
+    "start": "nodemon --",
+    "test:e2e": "playwright test --config cli-e2e-test.config.js"
+  },
+  "nodemonConfig": {
+    "exec": "bin/scaffolder-cli",
+    "ext": "ts",
+    "watch": "./src"
+  },
+  "dependencies": {
+    "@backstage/errors": "workspace:^",
+    "@backstage/plugin-scaffolder-node": "workspace:^",
+    "@backstage/plugin-scaffolder-react": "workspace:^",
+    "@backstage/types": "workspace:^",
+    "chalk": "^5.3.0",
+    "commander": "^12.1.0",
+    "fs-extra": "^11.2.0",
+    "js-yaml": "^4.1.0"
+  },
+  "devDependencies": {
+    "@backstage/cli": "workspace:^",
+    "@playwright/test": "^1.32.3",
+    "@types/js-yaml": "^3.12.5",
+    "@types/node": "^16.7.13",
+    "chalk": "^4.0.0",
+    "nodemon": "^2.0.15",
+    "typescript": "^4.4.4"
```

Comment:
Don't think you have to provide a `typescript` version in these subpackages, as they're dealt with from the repository root.